adapters/arpae.js

```diff
@@ -0,0 +1,292 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+import { default as moment } from 'moment-timezone';
+import { acceptableParameters, convertUnits } from '../lib/utils';
+const request = baseRequest.defaults({timeout: REQUEST_TIMEOUT});
+
+exports.name = 'arpae';
+
+const ckanResourceID = 'a1c46cfe-46e5-44b4-9231-7d9260a38e68';
+const searchUrl = 'https://dati.arpae.it/api/action/datastore_search';
+
+exports.fetchData = function (source, cb) {
+  const yesterday = moment.tz(source.timezone).subtract(1, 'days').format('Y-M-D');
+  // not doing paging right now, so set a limit
+  const limit = 100 * acceptableParameters.length;
+  const queryUrl = `${searchUrl}?resource_id=${ckanResourceID}&filters={"reftime": "${yesterday}"}&limit=${limit}`;
```

Comment:
After some testing, it looks like this reftime gives the time AT the time passed, not the time SINCE. So basically when I put in 2017-10-10, I see only things at 2017-10-10T00:00:00 not since then. Do you see that too?

Reply:
You're absolutely right. I'll switch it back to SQL.